types/src/v0/traits.rs

```diff
@@ -686,11 +684,39 @@ pub trait SequencerPersistence: Sized + Send + Sync + Clone + 'static {
     ) -> anyhow::Result<()>;
     async fn migrate_consensus(
         &self,
-        migrate_leaf: fn(Leaf) -> Leaf2,
-        migrate_proposal: fn(
+        _migrate_leaf: fn(Leaf) -> Leaf2,
+        _migrate_proposal: fn(
             Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
         ) -> Proposal<SeqTypes, QuorumProposal2<SeqTypes>>,
-    ) -> anyhow::Result<()>;
+    ) -> anyhow::Result<()> {
+        tracing::info!("migrating consensus data");
+
+        self.migrate_anchor_leaf().await?;
+        self.migrate_da_proposals().await?;
+        self.migrate_vid_shares().await?;
+        self.migrate_undecided_state().await?;
+        self.migrate_quorum_proposals().await?;
+        self.migrate_quorum_certificates().await
```

Comment:
Can we gate this by versions(and the other parts we start storing the new types) so we can merge this and not worry about deploying it